README.md

```diff
@@ -1,239 +1,224 @@
-- Different currencies use different numbers of decimal positions which is easy to get confused. (It's not correct to assume that all currencies have two decimal places, or that the only exception is JPY; making such assumptions will make it hard to internationalize your code to new countries.) For this reason, it's ideal if the number of decimal places is part of the data type.
-- In various contexts (e.g., presenting the quantity to the end user), the fractionality needs to be brought back in somehow. For example, `Intl.NumberFormat` only knows how to format Numbers, and can't deal with an integer + exponent pair.
-- Sometimes, fractional cents need to be represented too (e.g., as precise prices).
+Many currencies tend to be expressed with decimal quantities. Although it’s possible to represent money as integer “cents”, this approach runs into a couple of issues:
+
++ There’s a persistent mismatch between the way humans think about money and the way it’s manipulated in the program, causing mental overhead for the programmer aware of the issue.
+  + Some programmers may not even be aware of this mismatch. This opens the door to rounding errors whose source is unknown. If calculations start to get more involved, the chance of error increases.
++ Different currencies use different numbers of decimal positions which is easy to get confused; the hack of working with integers may not work when working with multiple currencies. For instance, it’s not correct to assume that all currencies have two decimal places, or that the only exception is JPY (Japanese yen); making such assumptions will make it hard to internationalize code to new countries. For this reason, it’s ideal if the number of decimal places is part of the data type.
++ In various contexts (e.g., presenting a quantity to the end user), the fractionality needs to be brought back in somehow. For example, `Intl.NumberFormat` only knows how to format Numbers, and can’t deal with an integer-and-exponent pair.
++ Sometimes, fractional cents need to be represented too (e.g., as precise prices that occur, for instance, in stock trading or currency conversion).
 
 #### Sample code
 
-For example, to add up a bill with a number of items, and add sales tax, the `BigDecimal` alternative (described below) could allow the following:
+Add up the items of a bill, then add sales tax:
 
 ```js
 function calculateBill(items, tax) {
-  let total = 0m;
+  let total = new Decimal("0");
   for (let {price, count} of items) {
-    total += price * BigDecimal(count);
+    total = total.add(price.times(new Decimal(count)));
   }
-  return BigDecimal.round(total * (1m + tax),
-                          {maximumFractionDigits: 2, round: "up"});
+  return total.multiply(tax.add(1));
 }
 
-let items = [{price: 1.25m, count: 5}, {price: 5m, count: 1}];
-let tax = .0735m;
-console.log(calculateBill(items, tax));
+let items = [{price: "1.25", count: 5}, {price: "5.00", count: 1}];
+let tax = "0.0735";
```

Comment:
There'd have to be `new Decimal` called on these strings at some point, because they get `.times()` and `.add()` methods called on them inside `calculateBill`.

Reply:
There's a call to `new Decimal` in the function, defined just above. The idea was that `items` contains "plain" data (strings, number literals), such as what one might manually write or which might come from an external source such as a DB.

Reply:
> I don't see where `tax` is passed to `new Decimal` in the function above

Oops, you're right. Fixed now. Thanks!